docs/resources/meeting_conferences.md

```diff
@@ -0,0 +1,551 @@
+---
+subcategory: "Meeting"
+---
+
+# huaweicloud_meeting_conference
+
+Using this resource to book a conference within HuaweiCloud.
+
+~> Using duplicate conference IDs (in the Console) will make terraform resource state (behavior) inconsistent.
+
+## Example Usage
+
+### Book an Ordinary Conference using third-party application authorization
+
+```hcl
+variable "app_id" {}
+variable "app_key" {}
+variable "conference_topic" {}
+variable "meeting_room_id" {}
+variable "account_id" {}
+
+resource "huaweicloud_meeting_conference" "test" {
+  app_id  = var.app_id
+  app_key = var.app_key
+
+  topic           = var.conference_topic
+  meeting_room_id = var.meeting_room_id
+  start_time      = "2006-01-02 15:04"
```

Comment:
suggest to update the year to 2022 or use the [timestamp](https://www.terraform.io/language/functions/timestamp) function in the usage

Reply:
done.